golang/cmd/sqlboiler2other/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var (
+	basedir string
+	ext     string
+)
+
+func main() {
+	flag.StringVar(&basedir, "basedir", "", "directory of `.go` files")
+	flag.StringVar(&ext, "ext", "", "rename files to this extension")
+	flag.Parse()
+
+	pattern := filepath.Join(basedir, "*.go")
+	log.Println("globbing", pattern)
+	paths, err := filepath.Glob(pattern)
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, p := range paths {
+		log.Println("processing", p)
+		data, err := ioutil.ReadFile(p)
+		if err != nil {
+			log.Fatal(err, p)
+		}
+		os.Remove(p)
+
+		s := string(data)
+		parts := strings.SplitN(s, "var _ = `", 2)
+		if len(parts) == 1 {
+			continue
+		}
+		s = parts[1]
+		s = strings.TrimSpace(s)
+		s = strings.TrimSuffix(s, "`")
+		s = strings.Replace(s, "\n\n\n", "\n\n", -1)
+		parts = strings.Split(s, "`\nvar _ = `")
+
```

Comment:
Was thinking if your implementation/idea on generating non-Go source code using `//` (https://github.com/choonkeat/hellocrud/blob/master/golang/sqlboiler/js/templates/01_struct.tpl) is more natural and flexible in terms of `sqlboiler2other` than `var _`? Something along the lines of: 

```
// for each path
// read file

s := string(data)
// Remove generated package comments
s = strings.Replace(s, "// Code generated by SQLBoiler (https://github.com/volatiletech/sqlboiler). DO NOT EDIT.\n", "", -1)
s = strings.Replace(s, "// This file is meant to be re-generated in place and/or deleted at any time.\n", "", -1)
// Find index of start of actual content
start := strings.Index(s, "//")
// Slice content from start index
s = s[start+2 : len(s)]
// Remove all `//`
s = strings.Replace(s, "//", "", -1)

// write s to file
```

Reply:
👍 will use block comments